soroban-env-host/src/host/declared_size.rs

```diff
@@ -138,6 +138,7 @@ impl_declared_size_type!(ClaimableBalanceEntry, 120);
 impl_declared_size_type!(LiquidityPoolEntry, 160);
 impl_declared_size_type!(ContractCodeEntry, 64);
 impl_declared_size_type!(ConfigSettingEntry, 96);
+impl_declared_size_type!(ExpirationEntry, 36);
```

Comment:
Please also add a corresponding entry in the two tests below: `test_expected_size` and `test_declared_size`

Reply:
Good catch. Updated.